ush/load_fv3gfs_modules.sh

```diff
@@ -22,9 +22,11 @@ if [[ -d /lfs3 ]] ; then
 	module load module_base.jet 
 elif [[ -d /scratch1 ]] ; then
     # We are on NOAA Hera
+        source "$HOMEgfs/versions/hera.ver"
```

Comment:
See above, move this outside the elif-block

Reply:
A `$target` variable isn't know at this point (this runs first in a job) so I don't have the ability to set:
```
source "$HOMEgfs/versions/${target}.ver"
```
I could set target in each of the if-blocks and then source `$target.ver` afterwards in the script. Thoughts?

Reply:
Let's leave it like this for here, but reevaluate whether we need both machine-setup and this script when we repeat this in develop.

Reply:
Agreed, want to do this and other aspects of this update better in develop when we have time to ponder this more.